segments/now_playing.sh

```diff
@@ -109,6 +109,9 @@ run_segment() {
 }
 
 __process_settings() {
+  if [ -z "$TMUX_POWERLINE_SEG_NOW_PLAYING_MUSIC_PLAYER" ]; then
```

Comment:
The indentation looks off. Make sure hard tabs are used as the rest of this file.